### ISSUE TITLE
Extract with format minimal does not set defaults

### ISSUE THREAD

**Issue by disintegrator** [lingui/js-lingui#222]
When the extract format is set to `minimal`, the extract command is not correctly setting the default message for the source locale. It does however work fine when the format is set to `lingui`

Given:
```html
<Trans id="userpage.tabs.items">Items</Trans>
```
Command:
```
yarn lingui extract
```

Output (`locale/en/messages.json`):
```json
{
  "userpage.tabs.items": "userpage.tabs.items"
}
```

Expected:
```json
{
  "userpage.tabs.items": "Items"
}
```

Config including package versions (extract from my package.json):
```json
{
  "dependencies": {
    "@lingui/react": "^2.0.8",
    "react": "^16.4.0",
    "react-dom": "^16.4.0"
  },
  "devDependencies": {
    "@lingui/babel-preset-react": "^2.0.8",
    "@lingui/cli": "^2.0.8",
  },
  "lingui": {
    "fallbackLocale": "en",
    "sourceLocale": "en",
    "localeDir": "<rootDir>/locale",
    "format": "minimal",
    "srcPathIgnorePatterns": [
      "/node_modules/",
      "<rootDir>/\\.next/",
      "<rootDir>/dist/",
      "<rootDir>/geolite2-country/"
    ]
  }
}
```

**Comment by snegostup** [comment]
It can be fixed editing one string as I described in #199 (starting at "And this leads to another issue...").
@tricoder42 should I create a PR?

**Comment by disintegrator** [comment]
I agree with your suggested fixes in #199 @snegostup.

**Comment by tricoder42** [comment]
@snegostup Sure, go for it! 👍

**Comment by tricoder42** [comment]
Fixed and released in [v2.2.0](https://github.com/lingui/js-lingui/blob/stable-2.x/CHANGELOG.md#220-2018-07-04). Thanks to all involved!
